src/overlays/actors/ovl_Eff_Dust/z_eff_dust.h

```diff
@@ -5,12 +5,32 @@
 #include "global.h"
 
 struct EffDust;
+typedef void (*EffDustActionFunc)(struct EffDust *, GlobalContext *);
+typedef void (*EffDustDrawFunc)(Actor *, GlobalContext *);
 
 typedef struct EffDust {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x41C];
+    /* 0x014C */ f32 distanceTraveled[0x40]; //!< For each particle. Normalized. From 0.0f to 1.0f
```

Comment:
Generally we just use `// text` for inline comments like these
```suggestion
    /* 0x014C */ f32 distanceTraveled[0x40]; // For each particle. Normalized. From 0.0f to 1.0f
```

Reply:
If done this way, this comment will be used for documenting that member in doxygen. Also, should I change the `0x40` for `64`, since it is the amount of particles?

Reply:
I see, that makes more sense. Tho we aren't really using that type of comments anywhere else atm so I think it would be a bit weird to introduce only here. Also not sure this is worth it, but it probably deserves its own discussion.

And yeah, `64` is probably better than `0x40` since it's an amount like you said